KNOWN_ISSUES.md

```diff
@@ -21,6 +21,7 @@ Please report any other issue you may encounter by using the app help form.
   typically from another device), the local file permissions are set to
   755 / 644, which means they could be more accessible than they were before
   for group / others.
+* Preserve the original creation date of the files and does not apply the sync date as the creation date.
```

Comment:
You should write this sentence backwards as this file is about what's not working and not how it should be working.